_translator-files/po/de/Installation-for-Macintosh.po

```diff
@@ -71,15 +72,14 @@ msgid ""
 "**macOS Sierra (10.12) or lower:** [Download legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})\\\\\n"
 "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
 msgstr ""
-"**macOS Sierra (10.12) or früher:** [Downloade legacy Version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})\\\\\n"
+"**macOS Sierra (10.12) oder früher:** [Download legacy version]({{ site.download_root_link }}{{ site.download_file_names.mac-legacy }})\\\\\n"
```

Comment:
The text in square brackets should be translated.

Reply:
Translated and noted to check/correct other .po files